public/app/features/alerting/unified/components/MetaText.tsx

```diff
@@ -22,12 +22,12 @@ const MetaText = ({ children, icon, color = 'secondary', ...rest }: Props) => {
       // allow passing ARIA and data- attributes
       {...rest}
     >
-      <Span variant="bodySmall" color={color}>
+      <Text variant="bodySmall" color={color}>
```

Comment:
If we are keeping it like we had previously, `span` will be a `bodySmall` by default, so we can remove it from here and from any other places that use `span` with this variant 👍

Reply:
Should not the `span` element have a `body` variant? I think the variant `bodySmall` is used just in specific use cases so shouldn't we make the user select it when it is needed and not have it by default in `span` elements?

Reply:
Yeah I cant remember what the default should be, but I think we set it as `bodySmall` before, whichever is best though 😄